.gitignore

```diff
@@ -25,3 +25,4 @@ out/
 gradle-wrapper.jar
 wallet/prod
 wallet/.cxx
+wallet/cpp
```

Comment:
There should not be any files here.  If there are then we missed some other parts of deleting all the cpp source and submodules